lib/config.js

```diff
@@ -127,7 +127,8 @@ class Config {
       version,
       name,
       private: isPrivate,
-      publish: !!name
+      publish: !!name,
+      packageUrl: 'https://www.npmjs.com/package/' + name
```

Comment:
Please add a helper to the `/lib/npm` module to provide this url.

Reply:
Will do.